sound/soc/sof/intel/shim.h

```diff
@@ -185,6 +185,7 @@ struct sof_intel_dsp_desc {
 	u32 d0i3_offset;
 	u32 quirks;
 	enum sof_intel_hw_ip_version hw_ip_version;
+	int (*read_sdw_lcount)(struct snd_sof_dev *sdev);
```

Comment:
Right, so this commit message is clear on the reasons to introduce the callback abstraction (different registers on different platforms).

Reply:
no change then